docs/core/tools/sdk-errors/netsdk1149.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "NETSDK1149: Built-in WinRT support not provided in .NET 5+"
+description: How to reference a component with built-in WinRT support from an app that targets .NET 5 or later.
+author: tdykstra
+ms.author: tdykstra
+ms.topic: error-reference
+ms.date: 06/28/2021
+f1_keywords:
+- NETSDK1149
+---
+# NETSDK1149: Built-in WinRT support not provided in .NET 5+"
+
+NETSDK1149 indicates that you're trying to reference a component that requires WinRT in an application that targets .NET 5 or a later version. These .NET versions don't have built-in support for WinRT. The full error message is similar to the following example:
+
+> *\<Component name>* cannot be referenced because it uses built-in support for WinRT, which is no longer supported in .NET 5 and higher.  An updated version of the component supporting .NET 5 is needed.
+
+You can resolve this error by changing the application's Target Framework Moniker (TFM). For more information, see [Call Windows Runtime APIs in desktop apps](/windows/apps/desktop/modernize/desktop-to-uwp-enhance).
```

Comment:
The link didn't work for me in preview, it might just be because it was coming from the review subdomain though

Reply:
Yes,, that's the reason -- if you copy and paste it on the end of "docs.microsoft.com", it works.

Reply:
We should clarify that this is the way to resolve the error if the developer is calling Windows Runtime APIs. It's also possible they get this error when referencing a 3rd party WinRT component, and in this case they will need an updated version of the component supporting .NET 5, which can be generated using C#/WinRT: https://docs.microsoft.com/en-us/windows/uwp/csharp-winrt/